tests/yast2_cmd/yast_lan.pm

```diff
@@ -0,0 +1,30 @@
+ # SUSE's openQA tests
+ #
+ # Copyright © 2009-2013 Bernhard M. Wiedemann
+ # Copyright © 2012-2019 SUSE LLC
+ #
+ # Copying and distribution of this file, with or without modification,
+ # are permitted in any medium without royalty provided the copyright
+ # notice and this notice are preserved.  This file is offered as-is,
+ # without any warranty.
+ 
+ # Summary: yast lan in cli, creates, edits and deletes device and lists and
+ # shows details of the device
+ # Maintainer: Vit Pelcak <vpelcak@suse.cz>
+
+use base 'consoletest';
+use strict;
+use testapi;
+sub run {
+    select_console 'root-console';
+    assert_script_run 'zypper -n in yast2-network';
+    assert_script_run 'yast lan add name=vlan50 ethdevice=eth0';
+    assert_script_run 'yast lan list';
```

Comment:
maybe not only check if the command returns 0 but also excepted the output

Reply:
see http://open.qa/api/testapi/#_validate_script_output

Reply:
Fixed

Reply:
how ? I dont see any change

Reply:
Fixed now.
For yast lan edit I had to increase timeout to 60s as otherwise the test timed out.